frontend/src/modules/entitieslist/components/EntitiesList.tsx

```diff
@@ -353,15 +352,24 @@ export const EntitiesListBase: React.FC<InternalProps> = (
                                 />
                             );
                         })}
-                    </ul>
-                ) : (
+                    </ul>) : (
                     // When there are no results for the current search.
                     <h3 className='no-results'>
                         <div className='fa fa-exclamation-circle'></div>
                         No results
-                    </h3>
+                    </h3>)
+                }
+
+                {hasMore && (
+                    <div ref={sentryRef}>
+                        <SkeletonLoader
+                            key={0}
+                            items={props.entities.entities}
+                        />
+                    </div>
                 )}
-            </InfiniteScroll>
+
```

Comment:
Another nit: I'd remove that blank line, too.

Reply:
Ok